auction-server/src/state.rs

```diff
@@ -689,6 +692,16 @@ impl ChainStoreSvm {
         });
         Ok(res)
     }
+
+    pub async fn add_recent_prioritization_fee(&self, fee: u64) {
```

Comment:
I suggest to do one of the following:
- Use VecDeque
- Create a new circular buffer type with push method. pass capacity inside the new function of this type and do the removal logic in the push function of the type
- Use this: https://github.com/agerasev/ringbuf

Reply:
VecDeque for now